lib/src/widgets/chat/ds_select_menu.widget.dart

```diff
@@ -0,0 +1,103 @@
+import 'dart:convert';
+
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSSelectMenu extends StatelessWidget {
+  final DSAlign align;
+  final Map<String, dynamic> content;
+  final Function? onSelected;
+
+  const DSSelectMenu({
+    Key? key,
+    required this.align,
+    required this.content,
+    this.onSelected,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(top: 18.0),
+      child: Column(
+        children: _buildSelectMenu(),
+      ),
+    );
+  }
+
+  List<Widget> _buildSelectMenu() {
+    final List<Widget> children = [];
+
+    int count = 0;
+
+    content['options'].forEach(
+      (dynamic option) {
+        count++;
+
+        children.add(
+          GestureDetector(
+            onTap: () {
+              if (onSelected != null) {
+                Map<String, dynamic> payload = {};
+
+                if (option.containsKey('value')) {
+                  String type = option['type'];
+                  payload = {
+                    "type": type,
+                    "content": type.contains('json')
+                        ? jsonDecode(option['value'])
+                        : option['value']
```

Comment:
I suggest using a comma at this point. depending on the editor configuration, the indentation may break.